api/v1alpha1/helmchartproxy_types.go

```diff
@@ -84,6 +85,15 @@ type HelmChartProxySpec struct {
 	// +optional
 	ReconcileStrategy string `json:"reconcileStrategy,omitempty"`
 
+	// RolloutStepSize is an opt-in feature that defines a step size during the
+	// initial rollout of HelmReleaseProxies on matching clusters. Once all
+	// rolled out HelmReleaseProxy resources are ready=true, the next batch of
+	// HelmReleaseProxy resources are reconciled. If undefined, will default to
+	// creating HelmReleaseProxy resources for all matching clusters.
+	// e.g. an int (5) or percentage of count of total matching clusters (25%)
+	// +optional
+	RolloutStepSize *intstr.IntOrString `json:"rolloutStepSize,omitempty"`
```

Comment:
Is it possible and currently processed changes to this field during recomcile, i.e. reset condition in case of reduce or increase it value?